scio-jdbc/src/main/scala/com/spotify/scio/jdbc/package.scala

```diff
@@ -65,7 +65,8 @@ package object jdbc {
   case class JdbcReadOptions[T](connectionOptions: JdbcConnectionOptions,
                                 query: String,
                                 statementPreparator: PreparedStatement => Unit = null,
-                                rowMapper: ResultSet => T) extends JdbcIoOptions
+                                rowMapper: ResultSet => T,
+                                fetchSize: Option[Int] = None) extends JdbcIoOptions
```

Comment:
Instead of requiring the user to wrap `Int` in `Some` here, what about use `Int` directly with a default of `-1`, which skips setting the fetch size?

Reply:
+1, yes agree.